dev/docker/hive/Dockerfile

```diff
@@ -3,14 +3,33 @@
 # This software is licensed under the Apache License version 2.
 #
 
-FROM ubuntu:16.04
-LABEL maintainer="support@datastrato.com"
+FROM ubuntu:16.04 AS packages
 
 ARG HADOOP_PACKAGE_NAME
 ARG HIVE_PACKAGE_NAME
 ARG JDBC_DIVER_PACKAGE_NAME
 ARG DEBIAN_FRONTEND=noninteractive
 
+COPY packages /tmp/packages
+
+ENV JAVA_HOME=/usr/local/jdk
+ENV HIVE_HOME=/usr/local/hive
+ENV HADOOP_HOME=/usr/local/hadoop
+
+# hadoop
+RUN mkdir ${HADOOP_HOME}
+RUN tar -xz -C ${HADOOP_HOME} --strip-components 1 -f /tmp/packages/${HADOOP_PACKAGE_NAME} && rm -rf /tmp/packages/${HADOOP_PACKAGE_NAME}
```

Comment:
Why we need use `--strip-component` params?

Reply:
same reason as  https://github.com/datastrato/gravitino/pull/3268#discussion_r1593430541